test/e2e/deployment.go

```diff
@@ -207,6 +206,20 @@ func stopDeployment(c *clientset.Clientset, oldC client.Interface, ns, deploymen
 	}
 }
 
+func waitForReplicaSetsGone(c *clientset.Clientset, ns string, options api.ListOptions) error {
+	return wait.Poll(poll, 10*time.Second, func() (bool, error) {
```

Comment:
nit: The general recommendation is to use wait.ForeverTestTimeout rather than each test defining its own timeout.
Its set to 30 secs